bedrock/base/templates/macros.html

```diff
@@ -276,7 +276,7 @@
     {% set ios_link = firefox_ios_url('mozorg') %}
   {% endif %}
   {% set android_link = settings.GOOGLE_PLAY_FIREFOX_LINK %}
-  <section id="send-to-device" class="{% if include_logo %}logo {% endif %}{% if include_title %}title{% else %}no-title{% endif %}" data-key="{{ settings.MOZILLA_LOCATION_SERVICES_KEY }}" {% if spinner_color %}data-spinner-color="{{ spinner_color }}{% endif %}">
+  <section id="send-to-device" class="{% if include_logo %}logo {% endif %}{% if include_title %}title{% else %}no-title{% endif %}" data-countries="{{ send_to_device_countries() }}"{% if spinner_color %} data-spinner-color="{{ spinner_color }}{% endif %}">
```

Comment:
Not introduced in this PR, but easy enough to fix here. The final double quote should be moved inside the final `{% endif %}`. Currently an extra double quote is generated when `spinner_color` is false.